lib/js/src/rpc/enums/PredefinedLayout.js

```diff
@@ -231,6 +231,16 @@ class PredefinedLayout extends Enum {
         return PredefinedLayout._MAP.LARGE_GRAPHIC_ONLY;
     }
 
+    /**
+     * Get the enum value for WEB_VIEW.
+     * Custom root template allowing in-vehicle WebEngine applications with appropriate permissions to show the
+     * application's own web view.
+     * @returns {String} - The enum value.
+     */
```

Comment:
Please align with other libs. Take text of RPC spec.

Reply:
@SilvioDC-Ford fixed in https://github.com/smartdevicelink/sdl_javascript_suite/pull/264/commits/07b2d177e818f95d584fb8f67e3638558f667bec